app/client/src/store/actions.ts

```diff
@@ -61,4 +61,40 @@ export default {
       commit('setStatus', { task: 'network', data: 'submitted' });
     });
   },
+  async getStatus({ commit, state }: { commit: Commit, state: RootState }) {
+    await axios.post(
+      `${config.server_url}/status`,
+      { hash: state.projectHash },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+    ).then((response) => {
```

Comment:
Any error handling? Should it display an error if the request fails?

Reply:
Will leave that to be handled in the api service class